test/gtest/aca_test_ovs_l2.cpp

```diff
@@ -736,6 +746,12 @@ TEST(ovs_l2_test_cases, DISABLED_2_ports_CREATE_test_traffic_CHILD)
   ASSERT_EQ(overall_rc, EXIT_SUCCESS);
   overall_rc = EXIT_SUCCESS;
 
+  // monitor br-tun for arp request message
```

Comment:
since all the traffic test cases will need to have this monitoring thread running, it is possible to just add it to "int main(int argc, char **argv)" in aca_test_main.cpp?

Reply:
there are different places in arp/dhcp tests, one on "br-tun" and one on "br-int"